README.md

```diff
@@ -16,3 +16,19 @@ npm init scramjetorg/sequence
 ```
 
 the main file and package.json file will be created out of `templates/default`.
+
+## Running
+Open a terminal in the folder of the selected template, then use:
+
+```bash
+# If you run this sample on Self Hosted Hub, please start it with process adapter option:
+DEVELOPMENT=1 sth --runtime-adapter=process
```

Comment:
not sure if DEVELOPMENT flag is required here.... please verify it not checked already